packaging/debian/control

```diff
@@ -0,0 +1,38 @@
+Source: ansible-runner
+Section: admin
+Priority: optional
+Maintainer: Ansible <info@ansible.com>
+Build-Depends: debhelper (>= 11), dh-python, python-all, python-setuptools, python3-all, python3-setuptools, python-pytest, python3-pytest, python-pexpect (>= 4.5), python3-pexpect (>= 4.5), python-mock, python3-mock, python-psutil, python3-psutil, python3-yaml
+Standards-Version: 4.1.3
+Homepage: https://www.ansible.com
+X-Python-Version: >= 2.7
+X-Python3-Version: >= 3.5
+Vcs-Git: https://github.com/ansible/ansible-runner.git
+
+Package: python-ansible-runner
+Architecture: all
+Depends: ${python:Depends}, ${misc:Depends}
+Recommends: ansible (>= 2.1)
+Conflicts: python3-ansible-runner
+Description: interfaces with Ansible from other systems (Python 2)
```

Comment:
This first line of the description seems weird and not a complete sentence... is this a short description?

Reply:
Yeah.  The linting that the package builder does wants a _really_ short length.

Reply:
https://lintian.debian.org/tags/description-too-long.html

Reply:
There was also another complaint, about starting the short description with an article, of all things.  :moyai:

Reply:
Lets get @tvo318 to rewrite this into something better.